S23/fulvius31/.gitattributes

```diff
@@ -0,0 +1,9 @@
+#
```

Comment:
Slightly painful to have to navigate through all these accessory files to get to the actual lowercasing code.  Sometimes nice to do a PR with just the reviewable code.